frontend/.env.example

```
@@ -1,5 +1,5 @@
 # Gemini API Configuration
 REACT_APP_GEMINI_API_KEY=your-gemini-api-key-here
-
+VITE_WS_URL=ws://localhost:8888
 # Optional: Model configuration
 # REACT_APP_GEMINI_MODEL=gemini-2.0-flash-exp
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)